### ISSUE TITLE
samples/guestbook/gcp: provision-db.sh depends on jq

### ISSUE THREAD

**Issue by ijt** [google/go-cloud#220]
When I ran `terraform apply` from the sample README, it failed like this on my Mac:

```
provision-db.sh: line 63: jq: command not found                                                                             
```

Of course I could just `brew install jq` but it's a bit of a rough spot. Maybe it would be better to rewrite this script in Go so it can't fail this way.

**Comment by zombiezen** [comment]
This will become obsolete if we switch scripts to using Go as suggested in #257.

**Comment by zombiezen** [comment]
@ijt Can we close this out now?

**Comment by ijt** [comment]
Yes, this is fixed by PR #288.
